core/src/main/java/org/lflang/generator/GeneratorBase.java

```diff
@@ -219,8 +219,6 @@ public void doGenerate(Resource resource, LFGeneratorContext context) {
     }
 
     // Process target files. Copy each of them into the src-gen dir.
-    // FIXME: Should we do this here? This doesn't make sense for federates the way it is
-    // done here.
     copyUserFiles(this.targetConfig, context.getFileConfig());
```

Comment:
```suggestion
```
I think we should remove the call to `copyUserFiles` because not all targets support the `files` property...

Reply:
I did this in [57f29cc](https://github.com/lf-lang/lingua-franca/pull/2333/commits/57f29cc1d51a4105db723fc2c78bca6aee892bf7). Also, I found that `TypeScript` target also doesn't support `files` property. Let's see if this doesn't break any other tests...